cmd/kubeadm/app/componentconfigs/kubelet.go

```diff
@@ -231,3 +239,18 @@ func isServiceActive(name string) (bool, error) {
 	}
 	return initSystem.ServiceIsActive(name), nil
 }
+
+// MutateCgroupDriver can be called to set the KubeletConfiguration cgroup driver to systemd
+// Currently this cannot be as part of Default() because the function is called for
+// upgrades too, which can break existing nodes after a kubelet restart.
+func MutateCgroupDriver(cfg *kubeadmapi.ClusterConfiguration) {
+	kubeletConfig := cfg.ComponentConfigs[KubeletGroup]
+	raw := kubeletConfig.Get().(*kubeletconfig.KubeletConfiguration)
+
+	if len(raw.CgroupDriver) == 0 {
+		driver := "systemd"
+		klog.V(1).Infof("setting the KubeletConfiguration cgroupDriver to the kubeadm static default of %q", driver)
```

Comment:
```suggestion
		klog.V(1).Infof("setting the KubeletConfiguration cgroupDriver to %q", driver)
```
The difference between static and dynamic default is a kubeadm implementation detail, I don't think it is required to surface it to the users